src/elm/Pages/Schedules/Form.elm

```diff
@@ -0,0 +1,231 @@
+        ]
+
+
+{-| viewEnabledCheckbox : renders checkbox inputs for selecting enabled
+-}
+viewEnabledCheckbox : ScheduleForm -> Html Msg
+viewEnabledCheckbox enableUpdate =
+    section [ Util.testAttribute "" ]
+        [ div [ class "form-control" ]
+            [ strong []
+                [ text "State"
+                , span [ class "field-description" ]
+                    [ text "( "
+                    , em [] [ text "Disabled schdules will not be run" ]
+                    , text " )"
+                    ]
+                ]
+            ]
+        , div
+            [ class "form-controls", class "-stack" ]
+            [ radio (Util.boolToYesNo enableUpdate.enabled) "yes" "Enabled" <| OnChangeEnabled "yes"
+            , radio (Util.boolToYesNo enableUpdate.enabled) "no" "Disabled" <| OnChangeEnabled "no"
+            ]
+        ]
+
+
+viewSubmitButtons : Model msg -> Html Msg
+viewSubmitButtons schedulesModel =
+    div [ class "buttons" ]
+        [ viewUpdateButton schedulesModel
+        , viewCancelButton schedulesModel
+        , viewDeleteButton schedulesModel
+        ]
+
+
+viewUpdateButton : Model msg -> Html Msg
+viewUpdateButton schedulesModel =
+    button
+        [ class "button"
+        , onClick <| Pages.Schedules.Model.UpdateSchedule
+        ]
+        [ text "Update" ]
+
+
+viewDeleteButton : Model msg -> Html Msg
+viewDeleteButton schedulesModel =
+    let
+        scheduleDeleteConfirm =
+            "-schedule-delete-confirm"
+
+        scheduleDeleteLoading =
+            "-loading"
+
+        ( deleteButtonText, deleteButtonClass ) =
+            case schedulesModel.deleteState of
+                NotAsked_ ->
+                    ( "Remove", "" )
+
+                Confirm ->
+                    ( "Confirm", scheduleDeleteConfirm )
+
+                Deleting ->
+                    ( "Removing", scheduleDeleteLoading )
+    in
+    button
+        [ class "button"
+        , class "-outline"
+        , class deleteButtonClass
+        , Util.testAttribute "schedule-delete-button"
+        , onClick <| Pages.Schedules.Model.DeleteSchedule
+        ]
+        [ text deleteButtonText ]
+
+
+viewCancelButton : Model msg -> Html Msg
+viewCancelButton schedulesModel =
+    case schedulesModel.deleteState of
+        NotAsked_ ->
+            text ""
+
+        Confirm ->
+            button
+                [ class "button"
+                , class "-outline"
+                , Util.testAttribute "schedule-cancel-button"
+                , onClick <| Pages.Schedules.Model.CancelDeleteSchedule
+                ]
+                [ text "Cancel" ]
+
+        Deleting ->
+            text ""
+
+
+
+-- HELPERS
+
+
+{-| eventEnabled : takes event and returns if it is enabled
+-}
+eventEnabled : String -> List String -> Bool
+eventEnabled event =
+    List.member event
+
+
+schedulesDocsURL : String
+schedulesDocsURL =
+    "#TODO"
```

Comment:
could we fill this value with velaDocsUrl?

Reply:
I put https://go-vela.github.io/docs/usage/schedules/ this follows the same convention as the others